modules/ismp/pallets/pallet/src/lib.rs

```diff
@@ -394,6 +405,24 @@ pub mod pallet {
 	pub type ChildTrieRoot<T: Config> =
 		StorageValue<_, <T as frame_system::Config>::Hash, ValueQuery>;
 
+	/// Genesis settings
+	#[pallet::genesis_config]
+	#[derive(frame_support::DefaultNoBound)]
+	pub struct GenesisConfig<T: Config> {
+		/// Should unsigned transactions be enabled. Preferred mode for testnet to simplify relayer
+		pub enable_unsigned_transactions: bool,
+		/// Phantom data
+		#[serde(skip)]
+		pub _phantom: PhantomData<T>,
```

Comment:
Should be private

```suggestion
	 _phantom: PhantomData<T>,
```

Reply:
Don't think it can be since you build these from outside the module.

Reply:
Oh yeah GenesisConfig is no longer strongly typed